examples/siro_sandbox/sandbox_app.py

```diff
@@ -16,22 +16,18 @@
 flags = sys.getdlopenflags()
```

Comment:
This file is refactored to just contain the `SandboxDriver` class (which itself will get renamed shortly).